modules/_labs/select/react/lib/SelectBase.tsx

```diff
@@ -312,6 +312,9 @@ const SelectBase = ({
         ...(onOptionSelection
           ? {
               onClick: (event: React.MouseEvent) => {
+                // TODO: Figure out why this preventDefault call exists.
+                // Removing it doesn't have any obvious consequences,
+                // but need to do more careful testing.
                 event.preventDefault();
```

Comment:
My best guess as to why this is here is if the menu option is a link or a button without an explicit `type="button"` which would cause the browser to navigate (if a link) or submit a form (if a button). Ideally, `event.preventDefault()` is kept to a minimum and only for specific issues as we might be accidentally preventing valid [default actions](https://www.w3.org/TR/uievents/#default-action) from occurring.